packages/shared/src/hooks/notifications/useBookmarkReminder.spec.tsx

```diff
@@ -136,7 +137,7 @@ describe('useBookmarkReminder hook', () => {
   });
 
   it('should properly set to next Monday', async () => {
-    const { result } = renderHook(() => useBookmarkReminder(), {
+    const { result } = renderHook(() => useBookmarkReminder(post), {
```

Comment:
We usually add tests for `trackEvent` so if you had time it would be cool to add 🙏 (non blocking)

Reply:
Oh, right. Yeah, I think it is a good practice. Let's add that.